File: snippets/fsharp/System/String/Split/limit.fs
module limit

//<snippet1>
let name = "Alex Johnson III"

let subs = name.Split(null, 2)

let firstName = subs[0]
let lastName =
    if subs.Length > 1 then
        subs[1]
    else
        ""

// firstName = "Alex"
// lastName = "Johnson III"
//</snippet1>

File: snippets/csharp/System/String/Split/limit.cs
using System;

namespace Split
{
    class Limit
    {
        public static void ThreeArgs()
        {
            //<snippet1>
            string name = "Alex Johnson III";

            string[] subs = name.Split(null, 2);

            string firstName = subs[0];
            string lastName;
            if (subs.Length > 1)
            {
                lastName = subs[1];
            }

            // firstName = "Alex"
            // lastName = "Johnson III"
            //</snippet1>
        }
    }
}
